templates/audio-to-text/template.yaml

```diff
@@ -244,15 +244,11 @@ spec:
           description: This is GitOps manifest for ${{ parameters.name }}
           namespace: ${{ parameters.namespace }}
           rhtapNamespace: rhtap
```

Comment:
asides this cleanup pr, but we should rename those params mentions `rhtap`

Reply:
Yeah I forgot to rename the vars when I opened https://github.com/redhat-ai-dev/ai-lab-template/pull/42, so the values are not `rhtap` anymore but I will include the renaming as part of this PR!